src/v2/ion/i18nTransformer.js

```diff
@@ -120,7 +120,7 @@ const updateLabelForUiSchema = (remediation, uiSchema) => {
   Logger.info('\t remediationName: ', remediation.name);
   Logger.info('\t uiSchema: ', JSON.stringify(uiSchema));
 
-  const authenticatorType = remediation.relatesTo && remediation.relatesTo.value.type
+  const authenticatorType = remediation.relatesTo?.value.type
```

Comment:
I guess it doesn't hurt to do `?.value?.type`

Reply:
It won't hurt, but we're unnecessarily adding extra logic / bytes into the compiled code. We should be prudent & use this only when necessary IMHO.

Reply:
We need to make some guidance on when and when not to use it. probably introduce a lint to support these constraints

Reply:
Basically we'd assume when `relatesTo` exists it must have `relatesTo.value`. Ideally we shall check whether `relatesTo.value` exists before looking into `relatesTo.value.type` (no assumption on API response otherwise runtime error)
Hence I think the guidance would be also use `?.` op when looking for object properties.